persistence/src/main/java/com/epam/esm/model/Certificate.java

```diff
@@ -115,10 +117,10 @@ public Builder(Certificate certificate) {
 
         public static Certificate merge(Certificate to, Certificate from) {
             Builder builder = new Builder(to);
-            if (nonNull(from.name) && !from.name.trim().isEmpty()) {
+            if (nonNull(from.name) && !StringUtils.isBlank(from.name)) {
```

Comment:
method isBlank/isNotBlank performs != null check inside.
`StringUtils.isNotBlank(from.name)` 
should be enough in this case